opentelekomcloud/resource_opentelekomcloud_csbs_backup_policy_v1.go

```diff
@@ -221,28 +222,35 @@ func resourceCSBSBackupPolicyRead(d *schema.ResourceData, meta interface{}) erro
 			return nil
 		}
 
-		return fmt.Errorf("Error retrieving backup policy: %s", err)
+		return fmt.Errorf("error retrieving backup policy: %s", err)
 	}
 
 	if err := d.Set("resource", flattenCSBSPolicyResources(*backupPolicy)); err != nil {
-		return err
+		return fmt.Errorf("[DEBUG] Error saving resource to state for OpenTelekomCloud CSBS backup policy (%s): %s", d.Id(), err)
 	}
 
 	if err := d.Set("scheduled_operation", flattenCSBSScheduledOperations(*backupPolicy)); err != nil {
-		return err
+		return fmt.Errorf("[DEBUG] Error saving scheduled_operation to state for OpenTelekomCloud CSBS backup policy (%s): %s", d.Id(), err)
 	}
 
 	if err := d.Set("tags", flattenCSBSPolicyTags(*backupPolicy)); err != nil {
-		return err
+		return fmt.Errorf("[DEBUG] Error saving policy tags to state for OpenTelekomCloud CSBS backup policy (%s): %s", d.Id(), err)
 	}
 
-	d.Set("name", backupPolicy.Name)
-	d.Set("common", backupPolicy.Parameters.Common)
-	d.Set("status", backupPolicy.Status)
-	d.Set("description", backupPolicy.Description)
-	d.Set("provider_id", backupPolicy.ProviderId)
-
-	d.Set("region", GetRegion(d, config))
+	me := &multierror.Error{}
```

Comment:
You can skip `me` initialization, just `me := multierror.Append(nil, ...` instead